### DISCUSSION TITLE
When importing ODF data.txt format error is reported and the file format cannot be detected

### DISCUSSION THREAD

**Discussion by ddyyasfh** [mtex-toolbox/mtex#1815] [Ask Anything] [answered]
Hi developers,
When importing ODF data.txt format error is reported and the file format cannot be detected
[odf.txt](https://github.com/mtex-toolbox/mtex/files/11419132/odf.txt)

**Comment by ralfHielscher** [comment, marked as answer]
You can do the following
```matlab
odf = ODF.load('odf.txt','columnnames',{'phi1','Phi','phi2','intensity'},'CS',crystalSymmetry('m-3m'),'interp')
```
Ralf.

**Reply by maguag** [reply]
Hi,
Thanks for the codes, but i met some problems when dealing with ODF data,  the max intensity comes to 300, is that nomal?
The following  is my codes, result and data.
```matlab
odf = ODF.load('cp020o.txt','columnnames',{'phi1','Phi','phi2','intensity'}, ...
    'CS',crystalSymmetry('m-3m'),'interp');
figure 
plot(odf,'phi2',45*degree,'antipodal','linewidth',1.5,'contourf',0:5:20,'colorRange',[0,20],'minmax','ShowText','on');
mtexColorbar 
```
![cp020o](https://user-images.githubusercontent.com/48279761/237013375-53b58653-bac7-47c3-abd4-36b56f2f74ee.jpg)
[cp020o.txt](https://github.com/mtex-toolbox/mtex/files/11428095/cp020o.txt)

**Reply by maguag** [reply]
> add a WeChat？

\_MaguaG\_

**Reply by ralfHielscher** [reply]
If you check your data you have values close to 300. So what you are wondering about?

**Reply by maguag** [reply]
> If you check your data you have values close to 300. So what you are wondering about?
Thanks for your reply, I'm really sorry for that i did't check the data very carefully. But I remember that before I export the odf data from the texture analysing softwere of Bruker D8 XRD instrument, the max value shows 132. I have checked the values again ,It does have few values close to 300, maybe the softwere just didn`t take those values into account.
